app/javascript/pages/account/components/DOBFieldset.tsx

```diff
@@ -90,17 +90,17 @@ const validateNumber = (required: boolean, value: string, maxValue: number, erro
 
 const validateAge = (month: string, day: string, year: string) => {
   if (year.length < 4) return "dob:invalid"
-  if (dayjs(`${month}/${day}/${year}`, "M/D/YYYY").valueOf() > dayjs().valueOf())
-    return "dob:invalid"
+  const enteredDate = dayjs(`${month}/${day}/${year}`, "M/D/YYYY")
+  const minDate = dayjs("1/1/1900", "M/D/YYYY")
+
   if (
-    dayjs(`${month}/${day}/${year}`, "M/D/YYYY").valueOf() <
-    dayjs().subtract(117, "years").valueOf()
-  )
+    !enteredDate.isValid() ||
```

Comment:
@tallulahkay I found that a user can input 19+2 or 19e2 for example and that number would pass through validation. I added this isValid callback to ensure that we block any of those types of inputs. Would you mind taking a look at this?

Reply:
Cool!!